docs/topics/dokka-gradle-configuration-options.md

```diff
@@ -0,0 +1,871 @@
+[//]: # (title: Dokka Gradle configuration options)
+
+Dokka has many configuration options to tailor your and your reader's experience.
+
+Below are some examples and detailed descriptions for each configuration section. You can also find an example
+with [all configuration options](#complete-configuration) applied.
+
+See [Configuration examples](dokka-gradle.md#configuration-examples) for more details on where to apply configuration blocks and how.
```

Comment:
I'm a bit confused as to how the link's contents are different from the current page. Isn't this supposed to be the place with details and examples? Could we clarify this so that users can judge better whether to follow the link?